src/MobileUI/ViewModels/ProfileViewModels/ProfileViewModelBase.cs

```diff
@@ -75,8 +85,6 @@ public void Receive(AchievementTappedMessage message)
         ProcessAchievement(message.Value);
     }
 
-    public abstract Task Initialise();
```

Comment:
If this isn't abstract, and we're not overriding it in derived types, should the derived types declare these as `new`?

Reply:
Hey Matt,

Since it's being removed here, wouldn't `new` be unnecessary in the derived classes? Unless I'm misunderstanding the question here

Reply:
Sorry you're right. It's been replaced by the `_initialise` method. But the purpose of this was so that the method could be called on the derived types when they're cast as the base type.

Reply:
Ah I see. Should I do anything to reimplement Initialise() here or is this fine?

Reply:
Let's leave it for now. We can see what happens in the v3 UI.